martin/src/bin/martin-cp.rs

```diff
@@ -246,6 +270,10 @@ enum MartinCpError {
         "More than one source found, please specify source using --source.\nAvailable sources: {0}"
     )]
     MultipleSources(String),
+    #[error(
+        "{0} of bounding box '{1}' must fit into {2:?}. Please check that your bounding box is in the `min_lon,min_lat,max_lon,max_lat` format."
+    )]
+    InvalidBoundingBox(String, String, RangeInclusive<f64>),
```

Comment:
why strings?  that requires allocation. For .0, use `&'static str`, and for the second string - perhaps the copy of a bounding box (if possible?)

Reply:
I don't think this super matters, but good call